### PR TITLE
cm file reviews

### PR DESCRIPTION
**Pull request by nabobalis**
CM time.

I broke down and ran yapf (I undid some things tho) on `color_tables`.

### CODE REVIEW

**Comment by nabobalis** [comment]
THIS ALSO FIXES THE ISSUES WITH THE DOC BUILD.

File: docs/conf.py [hunk 1]
```diff
@@ -145,16 +145,16 @@
...
 linkcheck_anchors = False
 
 # This is added to the end of RST files - a good place to put substitutions to
 # be used globally.
 rst_epilog = """
 .. SunPy
-.. _SunPy: http://sunpy.org
-.. _`SunPy mailing list`: http://groups.google.com/group/sunpy
-.. _`SunPy dev mailing list`: http://groups.google.com/group/sunpy-dev
+.. _SunPy: s//sunpy.org
```
Comment (dpshelio) [inline docs/conf.py]: Is this right?

Comment (nabobalis) [reply docs/conf.py]: Not sure what happened here. I will fix that.

**Comment by nabobalis** [comment]
Maybe fixes most of the CI?

**Comment by nabobalis** [comment]
Thanks @dpshelio

**Pull request merged**